recipes-openxt/xenclient-input-daemon/xenclient-input-daemon_git.bb

```diff
@@ -12,7 +12,7 @@ SRC_URI = "git://${OPENXT_GIT_MIRROR}/input.git;protocol=${OPENXT_GIT_PROTOCOL};
 	   file://input-daemon.initscript \
 "
 
-CFLAGS_append += " -Wno-unused-parameter -Wno-deprecated-declarations "
+CFLAGS_append += " -Wno-unused-parameter -Wno-deprecated-declarations"
```

Comment:
Nit: Might as well remove the first one too then, `+=` will add a space.

Reply:
Ill just drop this change, its whitespace and I'd rather have a 1 line smaller diff :D